spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -570,6 +570,13 @@ def name_id_version(format_urn)
           requested_attributes: ['email'],
         )
       end
+
+      it 'correctly sets the request URL' do
```

Comment:
@jmhooper @zachmargolis so this test works (e.g. fails when we don't update the path in the concern) but **only** when the routes are reordered a little so that the `get` comes after the `post`. I think this has to do with the fact that RSpec looks for a match in the routes file but doesn't check the HTTP verb, so if the `get` comes first it actually sets the `request` path to `/api/saml/auth2021`, even though this controller action is handled by `/api/saml/authpost2021` when `post` is used.